tls/server-tls-uart.c

```diff
@@ -131,35 +132,65 @@ static int uartIOTx(WOLFSSL *ssl, char *buf, int sz, void *ctx)
     int sent;
     CbCtx_t* cbCtx = (CbCtx_t*)ctx;
 
-    printf("Write: In %d\n", sz);
-    
-    sent = write(cbCtx->portFd, buf, sz);
-#if 0
-    if (logIF && sent) {
-        logIF->writeRaw(logIF, buf, sent);
-    }
+#ifdef DEBUG_UART_IO
+    printf("UART Write: In %d\n", sz);
 #endif
+
+    sent = write(cbCtx->portFd, buf, sz);
     if (sent == 0) {
         sent = WOLFSSL_CBIO_ERR_WANT_WRITE;
     }
 
-    printf("Write: Out %d\n", sent);
+#ifdef DEBUG_UART_IO
+    printf("UART Write: Out %d\n", sent);
+#endif
+
     return sent;
 }
 
 
-int main(void)
+int main(int argc, char** argv)
 {
     int ret = -1, err;
     WOLFSSL_CTX* ctx = NULL;
     WOLFSSL* ssl = NULL;
     CbCtx_t cBctx;
     struct termios tty;
-    const char testStr[] = "Testing 1, 2 and 3\r\n";
     byte echoBuffer[100];
+    const char* uartDev = UART_DEV;
+
+    if (argc >= 2) {
+        uartDev = argv[1];
+    }
 
+    /* open UART file descriptor */
     XMEMSET(&cBctx, 0, sizeof(cBctx));
-    cBctx.portFd = -1;
+    cBctx.portFd = open(uartDev, O_RDWR | O_NOCTTY);
+    if (cBctx.portFd < 0) {
+        printf("Error opening %s: Error %i (%s)\n",
+            uartDev, errno, strerror(errno));
+        ret = errno;
+        goto done;
+    }
+    tcgetattr(cBctx.portFd, &tty);
+    cfsetospeed(&tty, B115200);
+    cfsetispeed(&tty, B115200);
+    tty.c_cflag = (tty.c_cflag & ~CSIZE) | (CS8);
+    tty.c_iflag &= ~(IGNBRK | IXON | IXOFF | IXANY| INLCR | ICRNL);
+    tty.c_oflag &= ~OPOST;
+    tty.c_oflag &= ~(ONLCR|OCRNL);
+    tty.c_cflag &= ~(PARENB | PARODD | CSTOPB);
+    tty.c_lflag &= ~(ICANON | ECHO | ECHOE | ISIG);
+    tty.c_iflag &= ~ISTRIP;
+    tty.c_cc[VMIN] = 0;
+    tty.c_cc[VTIME] = 5;
+    tcsetattr(cBctx.portFd, TCSANOW, &tty);
+
+    /* Flush any data in the RX buffer - sure there is a better way to "flush" */
```

Comment:
Might want to remove `Sure there is a better way to "flush" `

...doesn't really add that much value.